pkg/ottl/contexts/ottldatapoints/datapoints.go

```diff
@@ -434,16 +431,14 @@ func accessTimeUnixNano() ottl.StandardGetSetter[TransformContext] {
 func accessDoubleValue() ottl.StandardGetSetter[TransformContext] {
 	return ottl.StandardGetSetter[TransformContext]{
 		Getter: func(ctx TransformContext) interface{} {
-			switch ctx.GetDataPoint().(type) {
-			case pmetric.NumberDataPoint:
+			if _, ok := ctx.GetDataPoint().(pmetric.NumberDataPoint); ok {
```

Comment:
Wouldn't it be better to assign the converted data point to a value so we don't have to convert it again on the following line?

Reply:
Yes